python/run-tests.py

```diff
@@ -22,22 +22,22 @@
 from os import path
 
 
-def test(root_dir, package):
-    # Run all of the test under test/python directory, each of them
-    # has main entry point to execute, which is python's unittest testing
+def test(root_dir, code_dir, packages):
```

Comment:
This used to run delta connect tests in https://github.com/delta-io/delta/commit/5c608fa108dacffe47cfb93ea9eeeda5080ea357